tests/backends/sqlite/test_parallel.py

```diff
@@ -0,0 +1,49 @@
+import shutil
+import tempfile
+from pathlib import Path
+
+from django.db import connections
+from django.test import TestCase
+
+
+class SQLiteParallelCloneTests(TestCase):
```

Comment:
This is missing the skipUnless decorator you see in the other files in this directory.